mkdocs.yml

```diff
@@ -13,6 +13,7 @@ nav:
   - Getting started: getting-started.md
   - Using Deno: using-deno.md
   - Dynamic modules: ng-module.md
+  - Nix build: nix-develop.md
```

Comment:
Our horizontal navigation is already large: 
![image](https://user-images.githubusercontent.com/7820865/114553908-83e92e80-9c3c-11eb-9d49-7edff877e1c9.png)

Following @brotzeit suggestion, we would need to change that to something like:

```yaml
nav:
    - Home:
      - Overview: README.md
      - Build / Develop: nix-develop.md
   ...
```

Reply:
Similar to [this](https://github.com/emacs-lsp/lsp-mode/blob/master/mkdocs.yml#L7-L26)

Reply:
I see